lib/api/controllers/bulkController.js

```diff
@@ -57,6 +58,11 @@ class BulkController extends BaseController {
     assertHasBody(request);
     assertBodyHasAttribute(request, 'bulkData');
 
+    // @todo pass directly body.bulkData
+    // if (!_.isPlainObject(request.body.bulkData)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACommentedCode)